doc/sphinx/source/projects/ad41101_iio/ad41101_iio.rst

```diff
@@ -0,0 +1,106 @@
+AD4110-1 IIO Application
```

Comment:
Add this rst file path to the projects/index.rst file to get reflected in the FW applications index.

Reply:
It is added already

Reply:
oh got it.. the issue I see was because of the typo on that line